src/main/resources/xml/schema/2020-1/nordic2020-1.opf.sch

```diff
@@ -50,8 +50,8 @@
             <assert test="parent::opf:package/@unique-identifier = dc:identifier/@id">[opf3a] the id of the dc:identifier must equal the value of the package elements unique-identifier
                 attribute</assert>
 
-            <assert test="count(dc:title[@id='maintitle']) = 1 or count(dc:title) = 1">[opf3b] exactly one dc:title must be present in the package document when no main title is specified.</assert>
-            <assert test="count(dc:title[@id='maintitle']) = 0 or count(dc:title) > 1">[opf3b] more than one dc:title must be present in the package document when a main title is specified.</assert>
+            <let name="main-title-id" value="translate(opf:meta[text() = 'main' and @property = 'title-type']/@refines, '#', '')"/>
```

Comment:
If there are more than one title-type=main, I think this XPath will give an error, since `translate()` expects a string as input.. You could add a `[1]`  after `opf:meta[…]`, but then we would not handle cases where there are multiple titles with title-type=main (it's quite unlikely to be more than one, but better safe than sorry).

It could be reordered like this:

```xml
<let name="main-title-id" value="opf:meta[text() = 'main' and @property = 'title-type']/translate(@refines, '#', '')"/>
```

This would return a sequence of strings, in case there are multiple main titles. The `count(dc:title[@id=$main-title-id]) = 1` expression in the assertion would then potentially match multiple main titles, and would fail if there are more than one main title (which is what we want).